Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -83,24 +106,52 @@ Function Upgrade-LIS ($LISTarballUrlOld, $LISTarballUrlCurrent, $allVMData , $Te
-                        Add-Content -Value "Old lis: $OldlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
-                        Add-Content -Value "New lis: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
-                        $sts = Check-Modules
-                        if( -not $sts[-1]) {
-                            Write-LogErr "Failed due to either modules are not loaded or version mismatch"
+                    $upgradedlismoduleVersion = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort -username "root" -password $password -command "modinfo hv_vmbus|grep -w `"version:`""
+                    if (Check-MinorLisVersionUpgrade -LISTarballUrlOld $LISTarballUrlOld -LISTarballUrlCurrent $LISTarballUrlCurrent) {
+                        if ($OldlismoduleVersion -eq $upgradedlismoduleVersion) {
+                            Write-LogInfo "LIS upgraded to `"$LISTarballUrlCurrent`" successfully"
+                            Write-LogInfo "Old lis: $OldlisVersion"
+                            Write-LogInfo "New lis: $upgradedlisVersion"
+                            Add-Content -Value "Old lis: $OldlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
+                            Add-Content -Value "New lis: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
+                            $sts = Check-Modules
+                            if( -not $sts[-1]) {
+                                Write-LogErr "Failed due to either modules are not loaded or version mismatch"
+                                return $false
+                            }
+                            return $true
+                        }
+                        else {
+                            Write-LogErr "LIS upgradation failed"
+                            Write-LogInfo $OldlisVersion
+                            Write-LogInfo $upgradedlisVersion
+                            Add-Content -Value "Old lis: $OldlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
+                            Add-Content -Value "New lis: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
                             return $false
                         }
-                        return $true
                     }
                     else {
-                        Write-LogErr "LIS upgradation failed"
-                        Add-Content -Value "Old lis: $OldlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
-                        Add-Content -Value "New lis: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
-                        return $false
+                        if ($OldlisVersion -ne $upgradedlisVersion) {
+                            Write-LogInfo "LIS upgraded to `"$LISTarballUrlCurrent`" successfully"
+                            Write-LogInfo "Old lis: $OldlisVersion"
```

Comment:
be consistent in format, correct version is LIS - all capital letters

Reply:
Changed everywhere to capital LIS